addons/website_event/models/event.py

```diff
@@ -38,6 +38,10 @@ class Event(models.Model):
              " of the event on the website.", copy=False)
     menu_id = fields.Many2one('website.menu', 'Event Menu', copy=False)
 
+    # if we are in this range of time, ticket can be buy on website
+    start_sale_date = fields.Datetime('Start date of sales')
```

Comment:
It seems those fields disappear in the final diff.

Very important message: your commits have to be complete. Don't introduce code in a commit that you remove or modify without notice in a further commit.